dev/com.ibm.ws.microprofile.config.1.3_fat/publish/servers/MapEnvVarServer/server.env

```diff
@@ -0,0 +1,5 @@
+MPCONFIG_FATTEST_ENV_VARIABLE=mpconfigtestValue
+__MPCONFIG_FATTEST_ENV_VARIABLE=__mpconfigtestValue
+MPCONFIG_FATTEST_ENV_VARIABLE__=mpconfigtestValue__
+MPCONFIG_FATTEST__ENV_VARIABLE=mpconfig__testValue
+MPCONFIGFATTESTENVVARIABLE=mpconfigtestValue
```

Comment:
don't you need some lower case and mixed case variables to test as well?

Reply:
and then some other non-alpha numerics (e.g. dot)

Reply:
so some things found on the 1st pass, some on the 2nd and some on the 3rd

Reply:
I have lots of interesting property names that get mapped to env vars in MapEnvVarServlet. I think that's the focus of the feature but I'll add a couple of new variables in server.env. I *think* they should work cross-platform.